gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -33,7 +33,10 @@
 
             // Built-in checks
             "ArrayEquals",
-            "MissingOverride");
+            "MissingOverride",
+            "MutableConstantField",
```

Comment:
Annoyingly the auto-fix for this actually results in quite a lot of lines going over the 120 char limit we have...

Given that this means more manual actions to merge one of these PRs (and might block baseline upgrades), could we actually just drop this one for now and keep the others?